test/test_files/function/gds/special_projected_graph.test

```diff
@@ -0,0 +1,57 @@
+-DATASET CSV tinysnb
+
+--
+
+-CASE SpecialProjectedGraph
+-STATEMENT CALL create_projected_graph('empty', [], []);
+---- ok
+-STATEMENT CALL weakly_connected_component('empty') RETURN *;
+---- 0
+-STATEMENT CALL k_core_decomposition('empty') RETURN *;
+---- 0
+-STATEMENT CALL page_rank('empty') RETURN *;
+---- 0
+-STATEMENT CALL create_projected_graph('n', ['person'], []);
+---- ok
+-STATEMENT CALL weakly_connected_component('n') RETURN node.ID, group_id;
+---- 8
+0|0
+2|1
+3|2
+5|3
+7|4
+8|5
+9|6
+10|7
+-STATEMENT CALL k_core_decomposition('n') RETURN node.ID, k_degree;
+---- 8
+0|0
+2|0
+3|0
+5|0
+7|0
+8|0
+9|0
+10|0
+-STATEMENT CALL page_rank('n') RETURN node.ID, rank;
+---- 8
+0|0.018750
+2|0.018750
+3|0.018750
+5|0.018750
+7|0.018750
+8|0.018750
+9|0.018750
+10|0.018750
+-STATEMENT CALL create_projected_graph('err', ['person'], ['studyAt']);
+---- error
+Binder exception: organisation is connected to studyAt but not projected.
+-STATEMENT CALL create_projected_graph('err', ['person'], 'studyAt');
+---- error
+Binder exception: studyAt has data type STRING. STRUCT or LIST was expected.
```

Comment:
The error message is confusing. You mean the input param's data type is mismatched? Is the following understand correct?
```
Input param 'studyAt' has unexpected data type STRING. We expect STRUCT or LIST as input.
```